src/sections/Careers/index.js

```diff
@@ -192,7 +192,7 @@ const Gsoc = () => {
                         <div className="grid">
                             <Row>
                                 <Col xs={12} sm={6} md={3}>
-                                    <Link to="/gsod">
+                                    <Link to="./gsod">
```

Comment:
Please revert any changes related to the careers page. Anyways all of these are going to be dynamic which I am working on currently and this will lead to merge conflict